airlabs-client/src/lib.rs

```diff
@@ -64,6 +64,15 @@ impl Client {
         self.get(request).await
     }
 
+    pub async fn flights(&self, params: api::FlightRequestParam) -> reqwest::Result<Response> {
```

Comment:
As you commented with my previous P/R, I think for this case, the enum is the best solution, as you wish to look for one code or another one.

Reply:
Yes, enum is a clean approach. I'd unified it with the request itself. Please have a look at the #13